cgi/user.pl

```diff
@@ -195,30 +195,32 @@
 		};
 
 		# Professional account
-		push @{$template_data_ref->{sections}}, {
-			id => "professional",
-			name => lang("pro_account"),
-			description => "if_you_work_for_a_producer",
-			note => "producers_platform_description_long",
-			fields => [
-				{
-					field => "pro",
-					type => "checkbox",
-					label => lang("this_is_a_pro_account"),
-					warning => sprintf(lang("this_is_a_pro_account_for_org"),"<b>" . $user_ref->{org} . "</b>"),
-					value => "off",
-				},
-				{
-					field => "pro_checkbox",
-					type => "hidden",
-					value => 1,
-				},
-				{
-					field => "requested_org",
-					label => lang("producer_or_brand") . ":",
-				}
-			]
-		};
+		if (defined $user_ref->{org}) {
```

Comment:
So that definitely will solve the warning, but then it will prevent users to create a professional account. When users create an account, $user_ref->{org} is not defined (yet), but we want to display them the checkbox option to say that it is a professional account.

The warning field should probably be removed, and instead, we can change the template to display the warning message, only  when needed.

Template is here : /templates/web/pages/user_form/user_form_page.tt.html

This part should be changed:
```

                    [% IF accepted_organization && section.id == "professional" %]
                        <p>[% field.warning %]</p>
                    [% ELSE %]
                        <label for="[% field.field %]">
                            <input type="checkbox" id="[% field.field %]" name="[% field.field %]" [% IF field.value == 'on' %]checked="checked"[% END %] />
                            [% field.label %]
                        </label>
                    [% END %]
```

the Lang.pm defines f_lang() and lang_sprintf() functions that can be used directly in the template.

Reply:
Thank you for reviewing! This is certainly a great approach to fix that warning.
I've pushed another commit with an update :)